packages/api/src/generated/schema.graphql

```diff
@@ -2058,9 +2058,8 @@ enum SubscribeErrorCode {
 }
 
 input SubscribeInput {
-  name: String
   subscriptionType: SubscriptionType
-  url: String
+  url: String!
```

Comment:
I think this is probably safe but technically an API change. Is there any case right now where the client would be sending a null URL?

Reply:
I think only the Web client would make this API call and it always sends a not-null `URL`